File: transform.py
from lxml import etree
import sys

# Transform files pased as asguments
print(sys.argv)

# Load XML & XSLT
xml = etree.parse(sys.argv[1])
xslt = etree.parse(sys.argv[2])

# Create tranformer
transform = etree.XSLT(xslt)

# Apply transform
resultado = transform(xml)

# Save result
with open("result.html", "wb") as f:
    f.write(etree.tostring(resultado, pretty_print=True, encoding="UTF-8"))

print("Transform completed. Load result.html")

File: transform.php
<?php

// Transform files pased as asguments
echo $argc;

// Load XML & XSLT
$xml = new DOMDocument();
$xml->load($argv[1]);

$xsl = new DOMDocument();
$xsl->load($argv[2]);


// Configure XSLT processor
$proc = new XSLTProcessor();
$proc->importStylesheet($xsl);

// Apply transform
$resultado = $proc->transformToXML($xml);

// Save result in HTML file
file_put_contents('result.html', $resultado);

echo "Transform completed. Load result.html\n";